### ISSUE TITLE
Update the README

### ISSUE THREAD

**Issue by cherryWood55** [cherryWood55/Quiz-Game#11]
1. Go through the files in the project.
2. Add instructions as to how the game is played in the README.
3. Submit a PR from your own branch to master.

**Comment by LiuSeeker** [comment]
Im on it

**Comment by LiuSeeker** [comment]
Issue already resolved

**Comment by cherryWood55** [comment]
> 
> 
> Issue already resolved

Yes it was resolved. However there are still changes which can be made. 
Would you like to go ahead?

**Comment by cherryWood55** [comment]
**People who are making a contribution to the project - Please add your names to the contributors list and send a PR**

**Comment by movy-niaj** [comment]
I'm on it.
I'm new to open source but I am quite good at Python(and C/bash/SQL)

**Comment by cherryWood55** [comment]
Okay. Go ahead.

**Comment by AmrDhaliwal** [comment]
Hi is this issue still open to contribute too?
